### ISSUE TITLE
Cannot build on latest hugo

### ISSUE THREAD

**Issue by 1ridic** [devcows/hugo-universal-theme#395]
```
Start building sites … 
hugo v0.120.1+extended linux/amd64 BuildDate=unknown

ERROR render of "term" failed: template: _internal/_default/rss.xml:3:9: executing "_internal/_default/rss.xml" at <site>: can't evaluate field email in type string
ERROR render of "section" failed: template: _internal/_default/rss.xml:3:9: executing "_internal/_default/rss.xml" at <site>: can't evaluate field email in type string
ERROR render of "taxonomy" failed: template: _internal/_default/rss.xml:3:9: executing "_internal/_default/rss.xml" at <site>: can't evaluate field email in type string
Built in 67 ms
Error: error building site: render: failed to render pages: render of "home" failed: template: _internal/_default/rss.xml:3:9: executing "_internal/_default/rss.xml" at <site>: can't evaluate field email in type string
```

ref: https://discourse.gohugo.io/t/render-of-section-failed-cannot-find-the-template-that-has-the-error/46883

**Comment by PatrickNByrne** [comment]
I ran into this as well. The fix in my case was to update my config.toml and remove the "author" key from the "params" section. Then, add a new root level section called "params.author" with keys for "name" and "email". 

Example:
```
[params.author]
name = "Your Name"
email = "admin@somesite.com"
```

**Comment by ryanfox1985** [comment]
it works fine in my end:

```
version: hugo v0.121.2+extended linux/amd64
```

```
> hugo server
Watching for changes in /home/guillermo/workspace/hugo-universal-theme/{archetypes,exampleSite,i18n,layouts,package.json,static}
Watching for config changes in /home/guillermo/workspace/hugo-universal-theme/exampleSite/config.toml
Start building sites … 
hugo v0.121.2+extended linux/amd64 BuildDate=unknown


                   | EN   
-------------------+------
  Pages            |  43  
  Paginator pages  |   0  
  Non-page files   |   0  
  Static files     | 105  
  Processed images |   0  
  Aliases          |  17  
  Sitemaps         |   1  
  Cleaned          |   0  

Built in 70 ms
Environment: "development"
Serving pages from memory
Running in Fast Render Mode. For full rebuilds on change: hugo server --disableFastRender
Web Server is available at http://localhost:1313/hugo-universal-theme/ (bind address 127.0.0.1) 
Press Ctrl+C to stop
```

**Comment by al3xandru** [comment]
This is with a clean clone:

```
WARN  calling IsSet with unsupported type "invalid" (<nil>) will always return false.
ERROR render of "page" failed: "~/themes/universal/layouts/_default/single.html:35:65": execute of template failed: template: _default/single.html:35:65: executing "_default/single.html" at <.Site.Params.date_format>: invalid value; expected string
ERROR render of "section" failed: "~/themes/universal/layouts/_default/list.html:59:75": execute of template failed: template: _default/list.html:59:75: executing "_default/list.html" at <.Site.Params.date_format>: invalid value; expected string
ERROR render: failed to render pages: render of "page" failed: "~/themes/universal/layouts/_default/single.html:35:65": execute of template failed: template: _default/single.html:35:65: executing "_default/single.html" at <.Site.Params.date_format>: invalid value; expected string
Rebuilt in 51 ms
```

**Comment by ryanfox1985** [comment]
still not reproduccing the error:

<img width="1055" alt="image" src="https://github.com/devcows/hugo-universal-theme/assets/1152728/70db18f1-c949-4a22-a5d7-54a26d5a71ce">

**Comment by Magneticdud** [comment]
PR #406 fixes the latest error due to that disqus deprecation, which happened one version earlier than what the message says

hopefully it can be merged soon

meanwhile, for a quick fix, copy those files from the PR in /layout to override the ones in the theme

**Comment by ryanfox1985** [comment]
closed by #406 , thanks @Magneticdud
